### PR TITLE
Added Multi Step Bar component

### PR DESCRIPTION
**Pull request by francoislg**
Component that is more verbose that the simple "StepProgressBar".

Used to show the Status of each step of a multi-step process.

I have most certainly done some bad things here, don't hesitate to throw all your comments at me :)

### CODE REVIEW

**Comment by MichelML** [comment]
@francoislg can you post a screenshot so we can have a visual idea of your feature, we have no live demo for vapor PRs

**Comment by francoislg** [comment]
image

File: scss/components/multi-step-bar.scss [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+.multi-step-bar-backdrop-container {
+  color: transparent;
+  pointer-events: none;
+  overflow: hidden;
+  .multi-step-bar {
+    transform: skewX(-20deg);
+    &:first-child {
+      padding-left: 10px;
```
Comment (MichelML) [inline scss/components/multi-step-bar.scss]: you can use the variable $default-margin everywhere here

File: scss/variables.scss [hunk 2]
```diff
@@ -504,3 +504,15 @@ $search-bar-spinner-border: 2px solid $medium-blue;
 $search-bar-spinner-border-radius: 50%;
 $search-bar-spinner-offset: 6px;
 $search-bar-spinner-size: 16px;
+
+// Multi Step Bar
+$multi-step-small-height: 2px;
+$multi-step-bar-text-padding: 8px 5px;
+$multi-step-bar-separation: 5px;
+$multi-step-sliding-animation-background-size: 28px;
```
Comment (MichelML) [inline scss/variables.scss]: you should name $multi-step-sliding-animation-background-size to $multi-step-sliding-animation-background-width and have another one $multi-step-sliding-animation-background-height of 50px too which you would use here https://github.com/coveo/vapor/pull/504/files#diff-e9e5407f26ebf88f698f422f2cd409a8R11

File: scss/components/multi-step-bar.scss [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  }
+}
+
+.multi-step-bar {
+  opacity: 1;
+  flex: 1;
+  display: flex;
+  align-content: center;
+  align-items: center;
+  min-height: 10px;
```
Comment (MichelML) [inline scss/components/multi-step-bar.scss]: we use variables for everything using pixel units, you should make a variable for that

**Pull request merged**